datafusion/core/src/physical_plan/aggregates/row_hash.rs

```diff
@@ -322,16 +322,6 @@ fn create_group_accumulator(
     }
 }
 
-/// Extracts a successful Ok(_) or returns Poll::Ready(Some(Err(e))) with errors
-macro_rules! extract_ok {
```

Comment:
This macro was removed, but I think breaking the main state loop change into two different functions makes is slightly harder to follow the logic.

Thus I am inclined not to pursue this PR anymore